src/Tools/BuildValidator/Program.cs

```diff
@@ -278,102 +278,92 @@ private static bool ValidateFiles(IEnumerable<AssemblyInfo> assemblyInfos, Optio
             LocalSourceResolver sourceResolver,
             LocalReferenceResolver referenceResolver)
         {
-            MetadataReaderProvider? pdbReaderProvider = null;
+            // Find the embedded pdb
+            using var originalPeReader = new PEReader(File.OpenRead(assemblyInfo.FilePath));
```

Comment:
>File.OpenRead(assemblyInfo.FilePath) [](start = 54, length = 36)

Does `PEReader.Dispose()` dispose of the `Stream`?

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.reflection.portableexecutable.pereader.-ctor?view=net-5.0#System_Reflection_PortableExecutable_PEReader__ctor_System_IO_Stream_

> Ownership of the stream is transferred to the PEReader upon successful validation of constructor arguments. It will be disposed by the PEReader and the caller must not manipulate it.